src/config/mod.rs

```diff
@@ -425,3 +592,65 @@ impl TestNodeConfig {
         }
     }
 }
+
+/// Account Generator
+/// Manages the generation of accounts for era-test-node
+#[derive(Clone, Debug, Deserialize)]
+pub struct AccountGenerator {
+    chain_id: u32,
+    amount: usize,
+    phrase: String,
+    derivation_path: Option<String>,
+}
+
+impl AccountGenerator {
+    pub fn new(amount: usize) -> Self {
+        Self {
+            chain_id: TEST_NODE_NETWORK_ID,
+            amount,
+            phrase: Mnemonic::<English>::new(&mut thread_rng()).to_phrase(),
+            derivation_path: None,
+        }
+    }
+
+    #[must_use]
+    pub fn phrase(mut self, phrase: impl Into<String>) -> Self {
+        self.phrase = phrase.into();
+        self
+    }
+
+    #[must_use]
+    pub fn chain_id(mut self, chain_id: impl Into<u32>) -> Self {
+        self.chain_id = chain_id.into();
+        self
+    }
+
+    #[must_use]
+    pub fn derivation_path(mut self, derivation_path: impl Into<String>) -> Self {
+        let mut derivation_path = derivation_path.into();
+        if !derivation_path.ends_with('/') {
+            derivation_path.push('/');
+        }
+        self.derivation_path = Some(derivation_path);
+        self
+    }
+
+    pub fn gen(&self) -> Vec<PrivateKeySigner> {
+        let builder = MnemonicBuilder::<English>::default().phrase(self.phrase.as_str());
+
+        let derivation_path = self.derivation_path.as_deref().unwrap_or("m/44'/60'/0'/0/");
```

Comment:
Stupid question, but why not the `DERIVATION_PATH` constant used in some other places?

Reply:
This should be the constant